### PR TITLE
Deploy prebuilds with travis

### PR DESCRIPTION
**Pull request by lgeiger**
Travis for the win 🎉 

Tested on my [fork](https://github.com/lgeiger/zmq-prebuilt/releases/tag/v1.2.3).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,3 +38,11 @@ before_install:
 install: env PKG_CONFIG_PATH=$ZMQ_PREFIX/lib/pkgconfig npm install
 
 script: travis_retry npm test
+
+deploy:
+  provider: script
+  skip_cleanup: true
+  script: node_modules/prebuild/bin.js --all -u $GH_TOKEN
```
Comment (lgeiger) [inline .travis.yml]: `prebuild` doesn't work inside a `npm` script. Thats why I'm using `node_modules/prebuild/bin.js`.

**Comment by rgbkrk** [comment]
Awesome! Next up is doing something with the appveyor builds!

File: .travis.yml (hunk 1 above)
Comment (rgbkrk) [inline .travis.yml]: 👍 good to clarify

You can comment inside this yml too.

File: .travis.yml [hunk 2]
```diff
@@ -17,8 +17,12 @@ os:
...
 
 env:
- - NODE_VERSION="5"
- - NODE_VERSION="4"
+
+  matrix:
+    - NODE_VERSION="5"
+    - NODE_VERSION="4"
+  global:
+    secure: AMY9chDNyNawxPhxAksCREhbQoOmphQKo2hDuYIPXBCUNF6Zik969QlB07TVjo+Bo0KVEbG4ANTdI0FkFIhg4MuJIWUkW30ZDyCSovUx8zyWB/7IN23A4Cx4p4ZnKjezTHUQpZ9InRIZzy9kOR+215IIBP6GfxbUY89eDA6ARElcv1rwDjZqSnLAxfie6v80vIbhWxNK4kQjiEyIez5eWBjRg4//dmb63TvJjc78GFTaqmh6lRIApAyA0uKka14XR/PJw5IpDR+0Q2omHl4t4zbu9MdVNYMSD+c6iNxOq3fqZJwnyJ1ELErbtR8oeKgZX1SOew5QTCUZWQsa+ySYTCzUnhWjhCgN6W0ciVtT2PslS/ZcvnrGGIxJAOmTZwINw8sAp8yYdY9QCJOGdB5ah2oUBLGKElcFGqSblohXUEM1tXeRchmpXiiAxnPNtxc6lkZzz6NxDl2OFXSmjMS4Cr8cA//Bn6gM04xRJyETy86/8KDoL+ENc7gkeiuYUOFqkAn5X8PrqPdsPmF7yA61IF1ru/OgtiuCh+eqnzZKBPImJo2KfUIqVuRi/6eUaDfWLT7d/J84Vbe3p2nPxoFd2pSw60f/lIkzFSRZx3vBjQiW2MVUN7ZTdawMOzE4TpIV7rDOkKe7qJJobquJEjd2X9Z+8JiTYXK2pPN2GceiqAM=
```
Comment (rgbkrk) [inline .travis.yml]: Since you were testing on your fork, does this need to be a key for this repo now?

Comment (lgeiger) [inline .travis.yml]: No I changed it.

**Comment by lgeiger** [comment]
I'll tackle appveyor after #33 is resolved.

**Comment by lgeiger** [comment]
Side note: Why is the mocha timeout 10 min? Isn't this a bit high?

**Comment by rgbkrk** [comment]
Yeah that seems excessively high. Might as well drop that to one minute.

**Pull request merged**